code/modules/paperwork/ticketmachine.dm

```diff
@@ -208,6 +225,7 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/ticket_machine, 32)
 		user.adjust_fire_stacks(1)
 		user.ignite_mob()
 		return
```

Comment:
Should emagged ones also call ``update_appearance()``? Maybe we can just remove this return if so.